.homeycompose/app.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "nl.zonneplan",
   "version": "1.0.21",
-  "compatibility": ">=5.0.0",
+  "compatibility": ">=12.2.0",
```

Comment:
is dit nodig?  dit is niet dashbaord.

Reply:
awesome , ik moet het even testen

Reply:
> is dit nodig? dit is niet dashbaord.

Ja, deze is nodig voor nieuwe Homey Energy support. Ga je ook nodig hebben als je de P1 en Zonnepanelen daar compatible mee gaat maken.

Reply:
p1 was al zichtbaar met homey , ze gebruiken alleen meter_power, measure power en homey moet je toevoegen als p1 (aan hun kant)

mijn zonnepanelen werken ook. maar ik ga het testen.